rs/ethereum/cketh/minter/src/eth_rpc.rs

```diff
@@ -661,7 +662,7 @@ pub async fn call<I, O>(
     method: impl Into<String>,
     params: I,
     mut response_size_estimate: ResponseSizeEstimate,
-) -> HttpOutcallResult<JsonRpcResult<O>>
+) -> Result<O, SingleCallError>
```

Comment:
I assume that `I` and `O` refer to the input and output, right?
This kind of makes sense but wouldn't it be easier to make sense of the letters if they referred to the actual types?

Reply:
> I assume that I and O refer to the input and output, right?
yes!

> This kind of makes sense but wouldn't it be easier to make sense of the letters if they referred to the actual types?

I'm not sure what you mean, since this method is generic the actual types used will change. Do you have a concrete example in mind?

Reply:
I would currently keep `I` and `O` since this is also used elsewhere (e.g., `sequential_call_until_ok`, `parallel_call` and also outside of ckETH).